src/query/functions/src/aggregates/aggregate_unary.rs

```diff
@@ -194,24 +205,19 @@ where
         _input_rows: usize,
     ) -> Result<()> {
         let column = T::try_downcast_column(&columns[0]).unwrap();
-        let column_iter = T::iter_column(&column);
         let state: &mut S = place.get::<S>();
         match validity {
             Some(bitmap) => {
```

Comment:
If all bits in the `bitmap` are true, we can also use `add_batch` to perform SIMD sum.